autoload/asterisk.vim

```diff
@@ -84,7 +84,7 @@ function! asterisk#do(mode, config) abort
         \   (a:mode isnot# 'n' ? "\<Esc>" : '')
         \   . 'm`'
         \   . (config.direction is s:DIRECTION.forward ? '0' : '$')
-        let aftermove = "\<C-o>"
+        let aftermove = '``'
```

Comment:
Can you try `"``\<C-o>"` in neovim?

Reply:
Jumps. Not work well.

Reply:
😢 ...

Reply:
Can you add if cond to change it only for neovim? It's impossible to support both Vim and NeoVim with same code without messing up jumplist history....